data/workflows.json

```diff
@@ -485,218 +639,254 @@
+                    "ocrd-cis-ocropy-denoise": "{\"level-of-operation\": \"page\", \"noise_maxsize\": 3.0, \"dpi\": 0}"
+                },
+                {
+                    "ocrd-tesserocr-deskew": "{\"operation_level\": \"page\", \"dpi\": 0, \"min_orientation_confidence\": 1.5}"
+                },
+                {
+                    "ocrd-tesserocr-segment": "{\"dpi\": 0, \"padding\": 4, \"shrink_polygons\": false, \"block_polygons\": false, \"find_tables\": true, \"find_staves\": false, \"sparse_text\": false, \"overwrite_segments\": true, \"segmentation_level\": \"region\", \"textequiv_level\": \"none\", \"overwrite_text\": true, \"raw_lines\": false, \"char_whitelist\": \"\", \"char_blacklist\": \"\", \"char_unblacklist\": \"\", \"tesseract_parameters\": {}, \"xpath_parameters\": {}, \"xpath_model\": {}, \"auto_model\": false, \"oem\": \"DEFAULT\"}"
+                },
+                {
+                    "ocrd-cis-ocropy-dewarp": "{\"dpi\": 0, \"range\": 4.0, \"smoothness\": 1.0, \"max_neighbour\": 0.05}"
+                },
+                {
+                    "ocrd-tesserocr-recognize": "{\"textequiv_level\": \"glyph\", \"overwrite_segments\": true, \"model\": \"Fraktur_GT4HistOCR\", \"dpi\": 0, \"padding\": 0, \"segmentation_level\": \"word\", \"overwrite_text\": true, \"shrink_polygons\": false, \"block_polygons\": false, \"find_tables\": true, \"find_staves\": false, \"sparse_text\": false, \"raw_lines\": false, \"char_whitelist\": \"\", \"char_blacklist\": \"\", \"char_unblacklist\": \"\", \"tesseract_parameters\": {}, \"xpath_parameters\": {}, \"xpath_model\": {}, \"auto_model\": false, \"oem\": \"DEFAULT\"}"
+                }
             ],
             "workflow_model": "Fraktur_GT4HistOCR",
+            "eval_tool": "ocrd-dinglehopper vNone",
             "document_metadata": {
-                "eval_workflow_url": "https://github.com/OCR-D/quiver-back-end/tree/main/workflows/ocrd_workflows/dinglehopper.txt",
-                "eval_data": "https://github.com/OCR-D/quiver-back-end/TODO",
                 "data_properties": {
                     "fonts": [
-                        "Antiqua"
+                        "Black Letter"
                     ],
-                    "publication_year": "16th century",
-                    "number_of_pages": 3,
+                    "publication_century": "1800-1900",
+                    "publication_decade": "",
+                    "publication_year": "19th century",
+                    "number_of_pages": 1,
                     "layout": "simple"
                 }
             }
         },
         "evaluation_results": {
             "document_wide": {
-                "wall_time": 41742,
-                "cer": 0.06721136853878373,
-                "cer_min_max": [
-                    0.03580246913580247,
-                    0.11059907834101383
-                ]
+                "wall_time": 28551,
+                "cpu_time": "(a few seconds)",
```

Comment:
Could we still provide a numeric value here like 0 or -1? On the front-end we could then map it to some string.

Reply:
This is what NextFlow gives me as a result, so unfortunately no.

Reply:
I think we might get numbers here if the process takes more time in general, but this is what we have for small run times.

Reply:
Problem is that it cannot be used for any dynamic calculations. Hm maybe I can map it on the front-end then.

Reply:
By chance I discovered that there seem to be real numbers buried in the NextFlow logs (and not in the result JSON where this string comes from). I will try to adjust the code but will move that objective to a separate branch.

Reply:
Done in the latest commit (1977ead)